lib/foreman/renderer/configuration.rb

```diff
@@ -60,6 +60,8 @@ class Configuration
         :match,
         :host_param_true?, :host_param_false?,
         :host_param, :host_param!,
+        :host_puppetserver,
+        :host_puppet_environment,
```

Comment:
Thinking about naming: should it be
```suggestion
        :host_puppet_server,
        :host_puppet_ca_server,
        :host_puppet_environment,
```

Reply:
why not, we don't need to follow the puppet naming I guess :))